s3-server-side-encryption/metadata.yml

```diff
@@ -9,4 +9,4 @@ tags:
   - "S3"
   - "Encryption"
 cloudProvider: "aws"
-terrafomVersions: "> 4.0.0"
+terraformVersions: "> 4.0.0"
```

Comment:
I see you just fixed typo, but this field is the version of the aws provider? How do we handle multi cloud that uses multiple providers?

Reply:
we have a ticket to do it so who ever will do it will think about it I just fixed typo